units/meson.build

```diff
@@ -103,7 +103,7 @@ units = [
         ['systemd-ask-password-wall.path',      '',
          'multi-user.target.wants/'],
         ['systemd-ask-password-wall.service',   ''],
-        ['systemd-boot-system-token.service',   'ENABLE_EFI',
+        ['systemd-boot-system-token.service',   'HAVE_GNU_EFI',
```

Comment:
there's also the new unit file `systemd-boot-update.service` which should be conditioned on HAVE_GNU_EFI, since it's very much sd-boot-specific

Reply:
This comment needs addressing.

Reply:
Is `systemd-boot-update.service` installed at all? I don't see it mentioned by any `meson.build` file and I've just compiled and installed systemd with `-Defi=true -Dgnu-efi=false` and it isn't installed.

Reply:
yeah, seems the hookup is missing

Reply:
#20791